model/operation.cc

```diff
@@ -555,6 +598,25 @@ namespace minsky
       if (auto g=group.lock())
         g->removeItem(*intVar);
   }
+  
+  float IntOp::x() const 
+  {
+    if (auto g=group.lock())
+      return zoomFactor()*m_x+g->x();
+    else {
+	  // adjust x() for coupled integral resizing operation. for feature 94.	
+	  if (coupled()) return intVar->m_x;	
```

Comment:
I don't see that this will do anything. Almost everything will be handled on line 605.

Reply:
I got rid of this method and use intVar->x() instead at appropriate places.